shared/src/Datadog.Trace.ClrProfiler.Native/instrumented_assembly_generator/instrumented_assembly_generator_helper.h

```diff
@@ -119,7 +119,9 @@ inline bool IsInstrumentedAssemblyGeneratorEnabled()
     {
         const auto isInstrumentedAssemblyGeneratorEnabled =
             shared::GetEnvironmentValue(cfg_instrumentation_verification_env);
-        if (isInstrumentedAssemblyGeneratorEnabled == WStr("1") ||
+        // default is true
+        if (isInstrumentedAssemblyGeneratorEnabled.empty() ||
```

Comment:
I think we should avoid duplicating this logic and instead reuse the `TryParseBooleanEnvironmentValue` method.

Reply:
Fix in d6e34e581c70cc2271806c12db2c3d4f2caf78f1